.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+### Description of Changes
+What has been changed?
+
+### Documentation
+Provide links to all relevant documentation that should be updated before asking for the PR to be reviewed. If this section is not relevant please write 'Not Applicable' rather than deleting the section.
+
+### Risks & Impacts
+The potential risks of this change as a bullet pointed list and how it might impact the wider application. Please consider how easily it would be to rollback these changes if they cause problems.
+
+### Security
+Are there any security considerations in this change? Has the way in which user input is handled been modified or extended? How could a determined attacker use this change to their advantage.
+
+### Testing
+How have the changes been tested and the potential risks mitigated? It's not sufficient to simply state 'test coverage'. Please ensure end to end tests have also been carried out.
+
+### Compare (For layered PRs)
+Generate compare URL from https://github.com/ebx/ebx-linkedin-sdk/compare so that it's easily accessible. This is ONLY REQUIRED FOR COMPLICATED, DEPENDENT OR LAYERED PRs. Feel free to delete this seciton if not required.
```

Comment:
```suggestion
Generate compare URL from https://github.com/ebx/ebx-linkedin-sdk/compare so that it's easily accessible. This is ONLY REQUIRED FOR COMPLICATED, DEPENDENT OR LAYERED PRs. Feel free to delete this section if not required.
```